pkg/monitoring/metrics/cdi-cloner/metrics.go

```diff
@@ -2,14 +2,10 @@ package cdicloner
 
 import (
 	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
-
-	runtimemetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
 )
 
 // SetupMetrics register prometheus metrics
 func SetupMetrics() error {
-	operatormetrics.Register = runtimemetrics.Registry.Register
-	operatormetrics.Unregister = runtimemetrics.Registry.Unregister
```

Comment:
Isn't the registerer in `operatormetrics` the same as the one in ctrl runtime? just trying to understand what really breaks here

Reply:
@machadovilaca can you please clarify the difference between the registerers?

Reply:
By default the `operatormetrics` uses the same registry as the Prometheus client. `controller-runtime` creates it's own custom registry, that's why controllers using the `controller-runtime` metrics endpoint need to do `operatormetrics.Register = runtimemetrics.Registry.Register`